plugins/experimental/traffic_dump/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+#######################
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+#  agreements.  See the NOTICE file distributed with this work for additional information regarding
+#  copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with the License.  You may obtain
+#  a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing permissions and limitations under
+#  the License.
+#
+#######################
+
+add_atsplugin(traffic_dump
```

Comment:
This depends on libswoc, the SSL library, and the ATS include path.

```cmake
target_include_directories(traffic_dump PRIVATE "${PROJECT_SOURCE_DIR}/include")

target_link_libraries(traffic_dump
    PRIVATE
        libswoc
        ${OPENSSL_SSL_LIBRARY}
)
```

Reply:
Sounds good.

As a side note, @JosiahWI and I looked at this together and noticed that these plugin builds inherit a lot of things (including the libswoc and openssl dependencies) from previous items in the root CMakeLists.txt file.

Reply:
Whoops, `${OPENSSL_SSL_LIBRARY}` should have quotes around t.